source/essences.textile

```diff
@@ -9,6 +9,30 @@ Essences are normal Rails models, so it is pretty easy to "add your own":create_
 
 endprologue.
 
+h3. Hints, Defaults, As Element Title
```

Comment:
The text is somehow confusing IMO. Maybe something like:

"Hints, Default Values and Element Titles"

What do you think?